pallets/delegation/Cargo.toml

```diff
@@ -14,10 +14,12 @@ substrate-wasm-builder-runner = {version = "3.0.0"}
 
 [dev-dependencies]
 ctype = {features = ["mock"], path = "../ctype", version = "1.1.1"}
-env_logger = {version = "0.8.4"}
 kilt-primitives = {default-features = false, path = "../../primitives"}
-pallet-balances = {branch = "polkadot-v0.9.12", default-features = false, git = "https://github.com/paritytech/substrate"}
+
```

Comment:
maybe we could also split the `#External dependencies` into two sections `# External dependencies` and `# Substrate dependencies`?

Reply:
I though it was too much and outscoped, but I see 🧐 -> https://github.com/KILTprotocol/mashnet-node/pull/295/commits/cf2059d968d39f3ac1d129de186d6ad6c180c96d

Reply:
yes I was actually about to say that. But I remembered that positive attitude